### PR TITLE
feat: [PIE-2733]: icon for pipeline variables in run pipeline form

### PR DESCRIPTION
**Pull request by AshwinBhatkal-Harness**
##### Summary:

Added icon for the pipeline variables section in run pipeline form.
This form is also being used in input set form, retry pipeline form and webhook pipeline input form. So the changes are visible for all the forms mentioned above

##### Jira Links:

https://harness.atlassian.net/browse/PIE-2733

##### Screenshots:

image

You can use the following comments to re-trigger PR Checks

- Jest: `retrigger jest`
- Prettier: `retrigger prettier`
- Type Check: `retrigger typecheck`
- ESLint: `retrigger eslint`
- Standards: `retrigger standards`
- Build: `retrigger build`
- Title Check: `retrigger titlecheck`
- Coverage: `retrigger coverage`
- Rebase: `trigger rebase`
- Cypress: `trigger cypress`
- Fix Prettier - `fix prettier`

### CODE REVIEW

**Comment by AshwinBhatkal-Harness** [comment]
trigger cypress

**Comment by AshwinBhatkal-Harness** [comment]
trigger cypress

**Comment by AshwinBhatkal-Harness** [comment]
trigger cypress

**Review by shamala-harness — APPROVED** [review]

File: src/modules/70-pipeline/components/PipelineInputSetForm/PipelineInputSetForm.tsx [hunk 1]
```diff
@@ -218,7 +218,16 @@ function PipelineInputSetFormInternal(props: PipelineInputSetFormProps): React.R
       ) : null}
       {template?.variables && template?.variables?.length > 0 && (
         <>
-          <div className={css.subheading}>{getString('customVariables.pipelineVariablesTitle')}</div>
+          <Layout.Horizontal spacing="small" padding={{ top: 'medium', left: 'large', right: 0, bottom: 0 }}>
```
Comment (chiragp1212) [inline src/modules/70-pipeline/components/PipelineInputSetForm/PipelineInputSetForm.tsx:221]: Please ensure this 'subheading' class is not applied to any other element in the file

**Review by chiragp1212 — APPROVED** [review]

File: src/modules/70-pipeline/components/PipelineInputSetForm/PipelineInputSetForm.tsx (hunk 1 above)
Comment (vkbansal) [reply src/modules/70-pipeline/components/PipelineInputSetForm/PipelineInputSetForm.tsx:221]: typecheck will take care of that

**Review by vkbansal — APPROVED** [review]

**Review by samarth-sharma31 — APPROVED** [review]

File: src/modules/70-pipeline/components/PipelineInputSetForm/PipelineInputSetForm.tsx (hunk 1 above)
Comment (AshwinBhatkal-Harness) [reply src/modules/70-pipeline/components/PipelineInputSetForm/PipelineInputSetForm.tsx:221]: It was used only in this file and here only. No other file imports the css for this file as well

**Review by vardanbansal-harness — APPROVED** [review]

**Pull request merged**